content/Rust-1.88.0.md

```diff
@@ -0,0 +1,109 @@
++++
+path = "2025/06/26/Rust-1.88.0"
+title = "Announcing Rust 1.88.0"
+authors = ["The Rust Release Team"]
+aliases = ["releases/1.88.0"]
+
+[extra]
+release = true
++++
+
+The Rust team is happy to announce a new version of Rust, 1.88.0. Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via `rustup`, you can get 1.88.0 with:
+
+```
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`](https://www.rust-lang.org/install.html) from the appropriate page on our website, and check out the [detailed release notes for 1.88.0](https://doc.rust-lang.org/stable/releases.html#version-1880-2025-06-26).
+
+If you'd like to help us out by testing future releases, you might consider updating locally to use the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report](https://github.com/rust-lang/rust/issues/new/choose) any bugs you might come across!
+
+## What's in 1.88.0 stable
+
+### Cargo automatic cache cleaning
+
+Starting in 1.88.0, Cargo will automatically run garbage collection on the cache in its home directory!
+
+When building, Cargo downloads and caches crates needed as dependencies. Historically, these downloaded files would never be cleaned up, leading to an unbounded amount of disk usage in Cargo's home directory. In this version, Cargo introduces a garbage collection mechanism to automatically clean up old files (e.g. `.crate` files). Cargo will remove files downloaded from the network if not accessed in 3 months, and files obtained from the local system if not accessed in 1 month. Note that this automatic garbage collection will not take place if running offline (using `--offline` or `--frozen`).
```

Comment:
hm. accessed by cargo or accessed by anyone or accessed by "that's a secret"?

Reply:
Accessed by cargo. It does not use atime to check usage by other tools if that's what you are referring to.